### PR TITLE
Merge development into loggers

### PR DESCRIPTION
**Pull request by iulianpascalau**
fixes after merge
resolved the new log lines inserted

### CODE REVIEW

File: process/block/preprocess/rewardTxPreProcessor.go [hunk 1]
```diff
@@ -149,10 +149,11 @@ func (rtp *rewardTxPreprocessor) RestoreTxBlockIntoPools(
 		strCache := process.ShardCacherIdentifier(miniBlock.SenderShardID, miniBlock.ReceiverShardID)
 		rewardTxBuff, err := rtp.storage.GetAll(dataRetriever.RewardTransactionUnit, miniBlock.TxHashes)
 		if err != nil {
-			log.Info(fmt.Sprintf("reward tx from mini block with sender shard %d and receiver shard %d, having %d txs, was not found in RewardTransactionUnit\n",
-				miniBlock.SenderShardID,
-				miniBlock.ReceiverShardID,
-				len(miniBlock.TxHashes)))
+			log.Debug("reward tx from mini block with was not found in RewardTransactionUnit",
```
Comment (bogdan-rosianu) [inline process/block/preprocess/rewardTxPreProcessor.go]: remove `with`

File: vm/systemSmartContracts/staking.go [hunk 2]
```diff
@@ -200,24 +200,20 @@ func (r *stakingSC) unBound(args *vmcommon.ContractCallInput) vmcommon.ReturnCod
...
 			"error", err.Error(),
 		)
 		return vmcommon.UserError
 	}
 
 	if registrationData.Staked == true || registrationData.UnStakedNonce <= registrationData.StartNonce {
-		log.Error("unBound is not possible for address with is staked")
-		JLS
-		log.Debug("validator did not unstaked yet")
+		log.Debug("unBound is not possible for address with is staked")
```
Comment (bogdan-rosianu) [inline vm/systemSmartContracts/staking.go]: please refactor this and include in the log also the second condition which could lead to an error

File: process/block/preprocess/rewardTxPreProcessor.go (hunk 1 above)
Comment (iulianpascalau) [reply process/block/preprocess/rewardTxPreProcessor.go]: Done

File: vm/systemSmartContracts/staking.go (hunk 2 above)
Comment (iulianpascalau) [reply vm/systemSmartContracts/staking.go]: rephrased

**Pull request merged**